src/h3lib/lib/bbox.c

```diff
@@ -117,5 +117,5 @@ int bboxHexRadius(const BBox* bbox, int res) {
     // any orientation of the GeoJSON encased in a circle defined by the
     // bounding box radius and center, it is guaranteed to fit in this k-ring
     // Rounded *up* to guarantee containment
-    return (int)ceil(bboxRadiusKm / (1.5 * centerHexRadiusKm));
+    return (int)ceil(bboxRadiusKm / (1.0 * centerHexRadiusKm));
```

Comment:
The comment above describes the purpose of the 1.5 constant, but I think there's some missing reasoning that gets from that comment to the revised code. It might be good to update the comment to reflect the conversation on Github about why/how we'r choosing to estimate this.

minor: Seems like this could be `(int)ceil(bboxRadiusKm / centerHexRadiusKm)`

Reply:
See edited comment and cleaned up code.